src/jit/lsra.cpp

```diff
@@ -8450,7 +8424,9 @@ void LinearScan::resolveRegisters()
         printf("Trees after linear scan register allocator (LSRA)\n");
         compiler->fgDispBasicBlocks(true);
     }
-    verifyFinalAllocation();
+
+// TODO(pdg): re-enable verification once it's moved to LIR, as it will almost certainly be useful.
+// verifyFinalAllocation();
```

Comment:
Note that this is also what produces the final dump of the allocation, which is essential for tracking down register allocation issues. Will this be addressed before merging?

---
In reply to: [75237385](https://github.com/dotnet/coreclr/pull/6689#discussion_r75237385) [](ancestors = 75237385)